scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -426,9 +426,13 @@ task ExtractTask {
     fi
 
     # This tool may get invoked with "Retry with more memory" with a different amount of memory than specified in
-    # the input `memory_gib`, so use the memory-related environment variables rather than the `memory_gib` input.
+    # the input `memory_gib`. If so, use the memory-related environment variables rather than the `memory_gib` input.
+    # But also be prepared if those memory-related variables are not set and fall back to using `memory_gib`.
     # https://support.terra.bio/hc/en-us/articles/4403215299355-Out-of-Memory-Retry
-    if [[ ${MEM_UNIT} == "GB" ]]
+    if [[ -z "${MEM_UNIT:-}" ]]
```

Comment:
`MEM_UNIT` and `MEM_SIZE` are currently not being set by the GCP Batch backend so I made these changes to do something reasonable so the integration tests can run.

Reply:
We should back these changes and those below out once the issue is addressed in Cromwell.